src/main/webapp/styles/general.css

```diff
@@ -4,6 +4,9 @@
   --backColor: white;
   --mainColor: #4285f4;
   --placeholder: #b7b7b7;
+  --gRed: #db4437;
+  --gLGray: #bdbdbd;
+  --gDGray: #666;
```

Comment:
This is just a question for my implementation of styling.  Can these global variables be used on any style sheet as long as general.css is imported on the same html page?

Reply:
yes

Reply:
just refer to them as var(--gLGray) for example